src/api/Completion.hx

```diff
@@ -5,9 +5,10 @@ typedef Completion = {
 	contents : String
 }
 
-enum CompletionType {
-	DEFAULT;
-	TOP_LEVEL;
+@:enum
+abstract CompletionType(Int) {
+	var DEFAULT = 0;
+	var TOP_LEVEL = 1;
```

Comment:
I changed this because when enum was used and I attempted to send completionType through context  I had run-time error "No such constructor", when completion was triggered